docs/root/configuration/operations/runtime.rst

```diff
@@ -274,6 +274,7 @@ The file system runtime provider emits some statistics in the *runtime.* namespa
 
   admin_overrides_active, Gauge, 1 if any admin overrides are active otherwise 0
   deprecated_feature_use, Counter, Total number of times deprecated features were used. Detailed information about the feature used will be logged to warning logs in the form "Using deprecated option 'X' from file Y".
+  deprecated_feature_seen_since_process_start, Gauge, Number of times deprecated features were used. This is not carried over during hot restarts.
```

Comment:
@rgs1 please add a release note for this. Thank you!

Reply:
Here you go: #11046